src/Hatchinq/Card.elm

```diff
@@ -0,0 +1,355 @@
+module Hatchinq.Card exposing (Config, Layout(..), Message(..), State, Thumbnail(..), View, configure, expandable, init, layout, update)
+
+{-|
+
+
+# Exposed
+
+@docs Config, Layout, Message, State, Thumbnail, View, configure, expandable, init, layout, update
+
+-}
+
+import Element exposing (Element, centerY, fill, px, shrink)
+import Element.Background as Background
+import Element.Border
+import Element.Font as Font
+import Hatchinq.Attribute as Attribute exposing (Attribute, custom, toElement, toInternalConfig)
+import Hatchinq.Button as Button
+import Hatchinq.IconButton as IconButton
+import Hatchinq.List exposing (roundImage)
+import Hatchinq.Theme as Theme exposing (Theme, icon, textWithEllipsis)
+import Hatchinq.Util exposing (takeFirstTwoLines)
+import Html
+import Html.Attributes
+
+
+
+-- TYPES
+
+
+{-| -}
+type alias Config msg =
+    { theme : Theme
+    , lift : Message msg -> msg
+    }
+
+
+type alias InternalConfig =
+    { expandable : Bool
+    , layout : Layout
+    }
+
+
+{-| -}
+type alias State =
+    { contentExpanded : Bool
+    }
+
+
+{-| -}
+type Layout
+    = MediaTop
+    | MediaCenter
+
+
+type alias Title =
+    { head : String
+    , subHead : Maybe String
+    }
+
+
+{-| -}
+type Thumbnail
+    = Icon String
+    | Image String
+
+
+defaultTheme =
```

Comment:
It means we can't change theme for buttons inside cards. Please, reuse theme that comes to the card